modules/migration-debugging-velero-admission-webhooks.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: ASSEMBLY
+[id="migration-debugging-velero-admission-webhooks"]
+= Issues with Velero and admission webhooks
+
```

Comment:
I think that removing this blank line and the following line might help.

Also, this file needs to be in a different directory than `modules` - putting it elsewhere will likely fix the includes and let you place it in the `_topic_map.yml` file.

Reply:
Different directory was the answer (I removed the extra lines also).